molecule/rbac-kafka-connect-replicator-kerberos-mtls-custom-rhel/molecule.yml

```diff
@@ -210,6 +227,35 @@ provisioner:
         zookeeper_kerberos_principal: "zookeeper/{{inventory_hostname}}.confluent@{{kerberos.realm | upper}}"
         zookeeper_kerberos_keytab_path: "{{ lookup('env', 'MOLECULE_SCENARIO_DIRECTORY') }}/keytabs/zookeeper-{{inventory_hostname}}.keytab"
 
+        kafka_connect_secret_registry_enabled: false
+
+        mask_secrets: false
+
+        oauth_enabled: true
+        ldap_with_oauth_enabled: true
```

Comment:
have we added any test for oauth only?

Reply:
currently
3/5 are oauth only
2/5 are ldap with oauth scenarios